src/Uno.UI.RuntimeTests/Extensions/TransformExtensions.Android.cs

```diff
@@ -6,22 +6,10 @@
 using Uno;
 using Uno.UI;
 
-namespace Windows.UI.Xaml.Media
+namespace Uno.UI.RuntimeTests.Extensions
 {
 	public static class TransformExtensions
 	{
-		internal static DependencyPropertyChangedEventArgs CreateInitialChangedEventArgs(this Transform transform, DependencyProperty property)
-		{
-			return new DependencyPropertyChangedEventArgs(
-				property,
-				null,
-				DependencyPropertyValuePrecedences.DefaultValue,
-				transform.GetValue(property),
-				transform.GetCurrentHighestValuePrecedence(property)
-			);
-		}
```

Comment:
I was looking for `DependencyPropertyChangedEventArgs` constructor usages and found this. This method was already unused so I deleted it.

The remaining method in this class is only used by runtime tests so I moved the class.